### PR TITLE
Storage Overview Section

### PR DESCRIPTION
**Pull request by mauerbac**
Screen Shot 2020-08-10 at 4 29 16 PM

### CODE REVIEW

File: docs/lib/storage/overview.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (renebrandel) [inline docs/lib/storage/overview.md:1]: If all of this is common, then we shouldn't need to have fragments.

File: docs/lib/storage/fragments/common/overview.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## Storage with Amplify
```
Comment (renebrandel) [inline docs/lib/storage/fragments/common/overview.md:1]: Title is not required if the page title says "Storage" "Concepts" already.

File: docs/lib/storage/fragments/common/overview.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+AWS Amplify Storage module provides a simple mechanism for managing user content for your app in public, protected or private storage buckets. The Storage category comes with built-in support for [Amazon S3 (Simple Storage Service)](https://docs.aws.amazon.com/AmazonS3/latest/dev/Welcome.html).
```
Comment (renebrandel) [inline docs/lib/storage/fragments/common/overview.md:3]: storage category

Comment (renebrandel) [inline docs/lib/storage/fragments/common/overview.md:3]: are we using uppercase "Storage" or lower case "storage"?

File: docs/lib/storage/fragments/common/overview.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## S3 Core Concepts 
+
+Amazon S3 stores data as objects within a container buckets. An object consists of a file and optionally any metadata that describes that file. To store an object in Amazon S3, you upload the file you want to store to a bucket. When you upload a file, you can set permissions on the object and any metadata.
+
+Buckets are the containers for objects. You can have one or more buckets. For each bucket, you can control access to it (who can create, delete, and list objects in the bucket), view access logs for it and its objects, and choose the geographical region where Amazon S3 will store the bucket and its contents.
+
+## Accessing AWS services
+
+With Storage, it's important to understand user level access to storage assets. While Amplify helps with abstraction with provisioning your S3 buckets, it is important to understand the right level of control for your customers.
```
Comment (renebrandel) [inline docs/lib/storage/fragments/common/overview.md:15]: uncertain: "user level" or "user-level"?

File: docs/lib/storage/fragments/common/overview.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+When you run amplify add storage, the CLI will configure appropriate IAM policies on the bucket using a Cognito Identity Pool Role. You will have the option of adding CRUD (Create/Update, Read and Delete) based permissions as well, so that Authenticated and Guest users will be granted limited permissions within these levels.
```
Comment (renebrandel) [inline docs/lib/storage/fragments/common/overview.md:17]: put `amplify add storage` in code snippet formatting

Comment (renebrandel) [inline docs/lib/storage/fragments/common/overview.md:17]: "Create, Update..."

File: docs/lib/storage/fragments/common/overview.md (hunk 3 above)
Comment (mauerbac) [reply docs/lib/storage/fragments/common/overview.md:3]: lowercase. thanks for catching

**Pull request merged**